### PR TITLE
Feat/no use libs

### PR DESCRIPTION
**Pull request by robrobbins**

### CODE REVIEW

File: build/swivel/Swivel.sol [hunk 1]
```diff
@@ -111,11 +111,12 @@ contract Swivel {
 
     IMarketPlace mPlace = IMarketPlace(marketPlace);
     address cTokenAddr;
-    address adapterAddr;
-    (cTokenAddr, adapterAddr) = mPlace.cTokenAndAdapterAddress(o.protocol, o.underlying, o.maturity);
+    // TODO if libraries work, we can revert this to only cTokenAddress if desired...
+    (cTokenAddr,) = mPlace.cTokenAndAdapterAddress(o.protocol, o.underlying, o.maturity);
+
+    // perform the actual deposit type transaction, specific to a protocol
+    require(deposit(o.protocol, cTokenAddr, principalFilled), 'deposit failed');
```
Comment (sourabhmarathe) [inline build/swivel/Swivel.sol:118]: Perhaps this is early optimization, but should we consider using `revert` and implementing our own errors from the get go?

**Review by sourabhmarathe — APPROVED** [review]
LGTM

File: build/swivel/Swivel.sol (hunk 1 above)
Comment (JTraversa) [reply build/swivel/Swivel.sol:118]: Should do all of them at once so we dont miss anything / end up against pattern (We said this on illuminate and i think one slipped through still)

**Comment by JTraversa** [comment]
Likewise LGTM, we'll jam on the read stuff in discord

**Review by JTraversa — APPROVED** [review]

**Pull request merged**